app/src/main/java/com/evanfuhr/pokemondatabase/interfaces/MoveDataInterface.java

```diff
@@ -12,4 +13,10 @@
     List<Move> getAllMoves(String nameSearchParam);
 
     Move getMoveByID(Move move);
+
+    Move getMoveMetaById(Move move);
```

Comment:
How is this method different from line 15 `getMoveByID`?

Reply:
There is a table called move_meta in the db that I found. This method is implemented here: https://github.com/evanfuhr/PokemonDatabase/pull/19/files#diff-88b11d4d5b18d26562e5df7a61c4c45aR199